rstudio/rstudio
# Feature/electron dmg background

### Intent

Add a background to the DMG window for the Electron Mac App

### Approach

Created a Background image with Retina support to be used in the DMG background. Icons were placed in proper place by design.

### Automated Tests

None

### QA Notes

Please build the desktop app, and run `yarn make` under `src/node/desktop`. Then, look `src/node/desktop/out/make` for a DMG file. Open it and the background located on `src/node/desktop/resources/background/dmg-background.tiff` should appear.

### Checklist


Addresses #10189
